code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

```diff
@@ -80,8 +88,8 @@
 		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right/unbreakable/nano)
 		)
 
-	heat_discomfort_strings = list("You feel too warm.")
-	cold_discomfort_strings = list("You feel too cool.")
+	heat_discomfort_strings = list("A sensor reminds you are nearing the  maximum safe operation tempature is below 553 Kelvin")
```

Comment:
"A sensor reminds you that your maximum safe operational temperature is below 553 Kelvin"
Replace with this, or something similar, what you've got here doesn't make sense, it has grammar and spelling issues.

Reply:
Yeah I'll fix spelling when I'm not sleepy. Sorry.

Reply:
No worries.

Reply:
Also those are currently nonfunctional. I think synthetic limbs overrides those strings but need more testing.

Reply:
Changed to 1.7. we will see after someone gets arrested.